Training5/task1/src/features/todos/TodoListItems.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { Checkbox, Button } from 'antd'
+import { selectedTodoByIds, todosDeleted, todosToggled,deleteTodo } from './todosSlice'
+
+
+const TodoListItems = ({ id }) => {
+
+    const todo = useSelector((state) => selectedTodoByIds(state, id))
```

Comment:
We should not use the selector to get todo here, why you do not pass using todo passed from prop? your todo and the todo passed from props are point to the same object, I don't think we need to run a loop for each todo